examples/jsm/renderers/webgpu/WebGPUNodeBuilder.js

```diff
@@ -0,0 +1,164 @@
+import WebGPUUniformsGroup from './WebGPUUniformsGroup.js';
+import { FloatUniform, Vector3Uniform } from './WebGPUUniform.js';
+
+import NodeSlot from '../nodes/core/NodeSlot.js';
+import NodeBuilder from '../nodes/core/NodeBuilder.js';
+
+class WebGPUNodeUniformsGroup extends WebGPUUniformsGroup {
```

Comment:
Can you also use just `WebGPUUniformsGroup`? Eventually, bindings should only be managed over the node material system.